### PR TITLE
linted and made changes to media

### PR DESCRIPTION
**Pull request by DJOcean**

### CODE REVIEW

**Comment by DJOcean** [comment]
I also made videos more responsive. the soundclouds look kinda large to me but im unsure how to get rid of that empty space since its an iframe

**Comment by DJOcean** [comment]
okay, totally just realized how to do that. once i make the changes can i just push it again or is it going to do that weird unable to merge thing?

File: index.html [hunk 1]
```diff
@@ -85,7 +85,7 @@
           <p>Whew! What a week : ) After an excellent concert Sunday by the Austin Saxophone Ensemble(featuring an excellent performance of American Counterpoint for Sax Quartet by Andy Wilds, Ben Kessler, Charlie Chadwell, and Nathan Mertens!), I had to double-time it writing and rehearsing guitar, piano, and string parts for a new song on an upcoming album, then took off Monday night, studio-bound. Tuesday and Wednesday flew by in the magical blur of recording, and we got some really good work done. Looking forward now to finishing up the album, and diving back into composition of a more classical-leaning nature!</p>
         </div>
         <div class="clearfix">
-          <h1>Poor Shadows of Elysium's present's Shakespeare's Antony & Cleopatra</h1>
+          <h1>Poor Shadows of Elysium's presents Shakespeare's Antony & Cleopatra</h1>
```
Comment (theWickedWebDev) [inline index.html]: this should be &amp; not &

**Comment by theWickedWebDev** [comment]
LGTM

**Pull request merged**